docs/apps/sole-factory.md

```diff
@@ -0,0 +1,45 @@
+---
+sidebar_position: 1
+title: Macrometa Sole Factory
```

Comment:
I'm confused by the naming. The first two seemed to be named based on features you are demonstrating, this one is based on what it is. Be consistent.